wcc-contentful-app/app/views/pages/show.html.erb

```diff
@@ -2,3 +2,6 @@
   next if section.nil? %>
   <%= render_section(section, index) %>
 <% end %>
+<% if @page.floatingCta.present? %>
```

Comment:
Since this only applies to Paper Signs, it's time to override this erb template in that project.  Otherwise, the next time that Microsites upgrades wcc-contentful, we'll start to see "NoMethodError" because `floatingCta` does not exist on the page object in that project.

Please close this PR and instead copy-paste this code into paper-signs as `app/views/pages/show.html.erb`